Model/Push/PartialPaymentProcessor.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Buckaroo\Magento2\Model\Push;
+
+use Buckaroo\Magento2\Api\PushProcessorInterface;
+use Buckaroo\Magento2\Api\PushRequestInterface;
+use Buckaroo\Magento2\Exception as BuckarooException;
+
+class PartialPaymentProcessor extends DefaultProcessor
+{
+
+    public function processPush(PushRequestInterface $pushRequest): bool
+    {
+        $this->pushRequest = $pushRequest;
+        $this->order = $this->orderRequestService->getOrderByRequest();
+        $this->payment = $this->order->getPayment();
+
+        // Check Push Dublicates
+        if ($this->receivePushCheckDuplicates()) {
+            throw new BuckarooException(__('Skipped handling this push, duplicate'));
+        }
+        // TODO: Implement processPush() method.
```

Comment:
remove comment?

Reply:
I deleted the entire class because was from an old version of push refactor.